src/bleanser/core/processor.py

```diff
@@ -22,8 +23,44 @@
 from plumbum import local # type: ignore
 
 
+# Mixin for helpers, this lets user subclass nicer
+class NormaliserHelpers:
```

Comment:
hmm maybe best to extract regular utility functions? imo mixin usually hints there is an extra state or something like that, and this only has static methods?

Reply:
Yeah I was considering both. It does mean more imports everywhere, but might be confusing as a mixin. I think youre right, will just make them standalone functions